library/src/scripts/textEditor/textEditorStyles.ts

```diff
@@ -32,13 +40,25 @@ export const textEditorClasses = useThemeCache(() => {
             height: viewHeight(90),
             backgroundColor: theme === "dark" ? colorOut("#1f2024") : colorOut(globalVars.elementaryColors.white),
             position: "relative",
+
+            $nest: {
+                "&& .monaco-editor": {
+                    paddingTop: unit(vars.editorPadding.padding.top),
+                    paddingLeft: unit(vars.editorPadding.padding.left),
```

Comment:
I'm pretty sure this is the cause of the editor menu being offset.

You'll need to add it to a container further outside the editor root like in the example here. https://wangchujiang.com/react-monacoeditor/